config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  # get "/customers", to: "customers#index"    
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id/items", to: "customer_items#create"
```

Comment:
These two routes look good. They are restful. I like your choice to send the items creation for a specific customer to the joins table controller.